CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+# Contributing guidelines
+
+We thank you in advance :thumbsup: :tada: for taking the time to
+contribute, whether with *code* or with *ideas*, to the Local EGA
+project.
+
+
+## Did you find a bug?
+
+* Ensure that the bug was not already reported by [searching under Issues].
+
+* If you're unable to find an (open) issue addressing the problem, [open a new
+  one]. Be sure to prefix the issue title with **[BUG]** and to include:
+
+  - a *clear* description,
+  - as much relevant information as possible, and
+  - a *code sample* or an (executable) *test case* demonstrating the expected behaviour that is not occurring.
+
+* If possible, use the following [template to report a bug].
+
+
+## Highlevel overview of how we work
+
+### Github projects
+
+We use the [Local EGA](https://github.com/orgs/EGA-archive/projects/3) github
+project in the EGA-Archive to manage our work.
+
+### How we use issues
+
+Issues are the central area for all work.
+
+### Standups
+
+Every day at 10:30 we have standups in Zoom meeting: <id>
+
+### Sprint Reviews and Planning
+
+Approximately every 2 weeks we have sprint reviews and then also a sprint
+planning meeting.
+
+
+
+## How to work on a new feauture/bug
+
+Create an issue on Github, and talk to the team members on the NBIS local-ega
+Slack channel. You can alternatively pick one already created.
+
+Assign yourself to that issue.
+
+Discussions on how to proceed about that issue take place in the comment
+section on that issue, beforehand.
+
+The keyword here is *beforehand*. It is usually a good idea to talk about it
+first. Somebody might have already some pieces in place, we avoid unnecessary
+work duplication and a waste of time and effort.
+
```

Comment:
> Once work on a feature/bug has been completed, make sure to provide helpful commit messages and create Pull Request. If a Pull Request tries to solve more than **one** issue it should probably be split into several PRs. 

I am thinking if we can make exception for bugs and allow a PR to contain more than **one** bug fix.

Reply:
Personally, I prefer several small ones to a few big ones. But as long as we don't reject pull requests because they are "too small" for some reason I am happy. I think it should be fine with a one-line pull request if it actually fixes a current bug or missing feature.

Reply:
I also prefer small PRs as they are usually easier to follow, but I assume the discussion is not if it is "too small" or not (it can be a small change with big impact), but if it is of value to the team and the project.

Reply:
A bug can be big and a feature can be small. What should be the upper limit size of one (suitable) PR can result in endless discussions. I also prefer small PRs and agree with @blankdots that such statement should be added. I think an issue of feature/bug should be able to be described by one or a few sentences.